tdrs-backend/tdpservice/parsers/parse.py

```diff
@@ -31,7 +31,7 @@ def parse_datafile(datafile):
         return errors
 
     # TODO: write a test for this line
-    case_consistency_validator = CaseConsistencyValidator(header, util.make_generate_parser_error(datafile, None))
+    case_consistency_validator = CaseConsistencyValidator(header, util.make_generate_parser_error(datafile, -1))
```

Comment:
Is there a reason why we should specify `-1` versus `None`?

Reply:
`None` was causing errors as `row_number` is a required field for `ParserError`. I'd be open to making it optional on the model